### ISSUE TITLE
Ensure that flight mode does the right thing for wifi

### ISSUE THREAD

**Issue by MerlijnWajer** [maemo-leste/bugtracker#190]
I don't think that flight mode actually turns off our wireless right now, we should figure out the right way to do that.

**Comment by parazyd** [comment]
Sidenote, I noticed Android allows turning on WiFi in Flight Mode.

**Comment by IMbackK** [comment]
thats right. ever since the actual flight mode EMI emmisions regulations have been relaxed "Flight Mode" is more of an artifact title for "cellular off"
